src/main/java/com/marmelab/quoridor/game/Game.java

```
@@ -135,6 +140,9 @@ private boolean isCrossable(final Pawn pawn) {
     }
 
     public void movePawn(final CardinalDirection direction) {
+        if (over) {
+            return;
```

If game is over, just fail silently to move pawn? I would like an exception better.